src/main/scala/com/lunatech/pointingpoker/actors/Room.scala

```diff
@@ -0,0 +1,116 @@
+package com.lunatech.pointingpoker.actors
+
+import java.util.UUID
+
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors}
+import akka.actor.{ActorRef => UntypedRef}
+import com.lunatech.pointingpoker.websocket.WSMessage
+import com.lunatech.pointingpoker.websocket.WSMessage.MessageType
+
+object Room {
+
+  sealed trait Command
+  final case class Join(user: User)                                     extends Command
+  final case class Leave(userId: UUID)                                  extends Command
+  final case class Vote(userId: UUID, estimation: String)               extends Command
+  final case class ClearVotes(userId: UUID)                             extends Command
+  final case class ShowVotes(userId: UUID)                              extends Command
+  final case class EditIssue(userId: UUID, issue: String)               extends Command
+  private[actors] final case class GetData(replyTo: ActorRef[Response]) extends Command
+
+  sealed trait Response
+  final case class DataStatus(data: RoomData) extends Response
+
+  final case class User(id: UUID, name: String, voted: Boolean, estimation: String, ref: UntypedRef)
+
+  final case class RoomData(users: List[User], currentIssue: String, issueLastEditBy: Option[UUID])
+  object RoomData {
+    val empty: RoomData = RoomData(List.empty[User], "", Option.empty[UUID])
+  }
+
+  def apply(roomId: UUID): Behavior[Command] =
+    Behaviors.setup[Command] { _ =>
```

Comment:
Do you need to call `setup` here? If I'm not mistaken, `receiveBehaviour(roomId, RoomData.empty)` should be sufficient.

Reply:
Same as the responses, I would like to keep the structure in case of growth.